polyfill.go

```diff
@@ -0,0 +1,174 @@
+package bitswap
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ipfs/go-bitswap/client"
+	"github.com/ipfs/go-bitswap/message"
+	"github.com/ipfs/go-bitswap/metrics"
+	"github.com/ipfs/go-bitswap/network"
+	"github.com/ipfs/go-bitswap/server"
+	"github.com/ipfs/go-bitswap/tracer"
+
+	"github.com/ipfs/go-block-format"
+	"github.com/ipfs/go-cid"
+	"github.com/ipfs/go-ipfs-blockstore"
+	"github.com/ipfs/go-ipfs-exchange-interface"
+	logging "github.com/ipfs/go-log"
+	"github.com/libp2p/go-libp2p-core/peer"
+
+	"go.uber.org/multierr"
+)
+
+var log = logging.Logger("bitswap")
+
+// old interface we are targeting
+type old interface {
```

Comment:
Can we just call this Bitswap?  (And the file `bitswap.go`?) Calling this a "polyfill" and "old" makes sense to us in the context of refactoring, but to folks without the historical context it's not a very useful naming scheme as this is the normal way most people will be using Bitswap.

Reply:
You think most people will keep using the polyfill interface in the future ? even for new software that will be written ?

Reply:
I mean there's plenty of evidence that lots of people use them both in tandem, outside of that I'm not going to try to predict the future. I know that if I see things called "polyfill" and "old" I'd be disinclined to use it, which is not the intention. It's a first class citizen that we intend on keeping around indefinitely--I think we should name it as such.

Reply:
> It's a first class citizen that we intend on keeping around indefinitely--I think we should name it as such.

I was not under this impression, but ok, I'll fix then.